test/com/facebook/buck/android/testdata/android_project/scala/com/sample/lib/BUCK.fixture

```diff
@@ -0,0 +1,35 @@
+android_library(
+  name = 'lib',
+  srcs = glob(['*.scala']),
+  language = 'SCALA',
```

Comment:
can we also add a test that says compilation will fail if the language parameter isn't something recognized? ("java", "scala",etc...)